styles/Home.module.css

```diff
@@ -146,3 +146,100 @@
     width: 100%;
   }
 }
+
+.navBar{
+  
+
+}
+.navBar > ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  display:flex;
+  background-color: #93D973;
+  align-items:center;
+  font-size: 30px;
+  
+  
+  
+}
+.navBar li {
+  display: block;
+
+  text-align: center;
+  padding: 0px 16px 0px 0px;
+  text-decoration: none;
+  
+  
+}
+
+
+.navBar Link {
+  
+  border: 10px solid
+}
+
+
+.navBar li a:hover {
+  background-color:#DB0A76
+}
+
+
+
+	.homeContainer {
+		display: grid;
+    gap: 10px;
+		grid-template-columns:repeat(4,minmax(200px,auto));
+		grid-template-rows: repeat(3,minmax(100px,auto));
+		background-color: #fff;
+    
+    
+	}
+
+	
+
+
+
+	.homeHeader{
+		grid-column: 1 / 4;
+    grid-row: 1;
+	}
+
+	.homeLogin {
+    grid-column: 4 ;
```

Comment:
Fix indentation for this file. Also please remove redundant blank lines if not removed by linter itself.